src/views/FarmAuction/hooks/useCurrentFarmAuction.ts

```diff
@@ -1,58 +1,41 @@
-import { useState, useEffect } from 'react'
+import { useEffect, useState } from 'react'
 import isEqual from 'lodash/isEqual'
+import useSWR from 'swr'
 import { useFarmAuctionContract } from 'hooks/useContract'
-import { Auction, ConnectedBidder, Bidder } from 'config/constants/types'
+import { ConnectedBidder } from 'config/constants/types'
 import { getBidderInfo } from 'config/constants/farmAuctions'
-import useLastUpdated from 'hooks/useLastUpdated'
 import { AUCTION_BIDDERS_TO_FETCH } from 'config'
-import { useFastRefreshEffect } from 'hooks/useRefreshEffect'
+import { FAST_INTERVAL } from 'config/constants'
 import { BIG_ZERO } from 'utils/bigNumber'
-import { sortAuctionBidders, processAuctionData } from '../helpers'
+import { processAuctionData, sortAuctionBidders } from '../helpers'
 
 export const useCurrentFarmAuction = (account: string) => {
-  const [currentAuction, setCurrentAuction] = useState<Auction | null>(null)
-  const [bidders, setBidders] = useState<Bidder[] | null>(null)
+  const { data: currentAuction = null } = useSWR(
+    ['farmAuction', 'currentAuction'],
+    async () => {
+      const auctionId = await farmAuctionContract.currentAuctionId()
+      const auctionData = await farmAuctionContract.auctions(auctionId)
+      return processAuctionData(auctionId.toNumber(), auctionData)
+    },
+    { refreshInterval: FAST_INTERVAL },
+  )
+
+  const { data: bidders = null, mutate: refreshBidders } = useSWR(
+    ['farmAuction', 'bidders'],
```

Comment:
check `currentAuction ? ['farmAuction', 'bidders'] : null` in cause currentAuction maybe null at first

Reply:
I will fix that

Reply:
Done